index.html

```diff
@@ -11003,6 +11086,76 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-textseparation">
+			<pdef>aria-textseparation</pdef>
+			<div class="property-description">
+				<p>Defines how text content of an <a>element</a> is separated from text content of neighboring elements.</p>
+		        <p>Authors SHOULD ensure that the type of text separation specified matches the visual rendering.</p>
+		        <p>The <pref>aria-textseparation</pref> <a>property</a> is an enumerated type. Any value not recognized in the list of allowed <span>values</span> SHOULD be treated by <a>assistive technologies</a> as if the value <code>inherit</code> had been provided. For future expansion, if the attribute is not present or its value is an empty string or <code>undefined</code>, the default value of <code>inherit</code> applies.</p>
+			</div>
+			<table class="property-features">
+				<caption>Characteristics:</caption>
+				<thead>
+					<tr>
+						<th scope="col">Characteristic</th>
+						<th scope="col">Value</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th class="property-related-head" scope="row">Related Concepts:</th>
+						<td class="property-related">
+							<ul>
+								<li>String concatenation, CSS rendering</li>
+							</ul>
+						</td>
+					</tr>
+					<tr>
+						<th class="property-applicability-head" scope="row">Used in Roles:</th>
+						<td class="property-applicability">Placeholder</td>
```

Comment:
I think we'd need to replace these placeholders with the tentative name of the role for this initial merge?

Reply:
The Placeholder gets replaced with any role(s) that list them as "supported".
Please see the table in the [aria-textseparation preview](https://pr-preview.s3.amazonaws.com/w3c/aria/pull/805.html#aria-textseparation).
It lists the `generic` role.

Reply:
Ok cool, thought so. Thanks!